types/localized-block-data.d.ts

```diff
@@ -1,49 +1,92 @@
 type RemoteDataBinding = Pick< RemoteDataResultFields, 'name' | 'type' >;
 type AvailableBindings = Record< string, RemoteDataBinding >;
 
+/**
+ * This corresponds directly to the input schema defined for a data source.
+ */
 interface InputVariable {
+	/** The display friendly name of the variable */
 	name: string;
+	/** Whether the variable is required */
 	required: boolean;
+	/** The slug of the variable */
 	slug: string;
+	/** The type of the variable */
 	type: string;
 }
 
+/**
+ * This corresponds directly to the overrides defined for a data source.
+ */
 interface InputVariableOverride {
+	/** The display friendly name of the variable */
 	display_name?: string;
+	/** The help text of the variable */
 	help_text?: string;
+	/** The slug of the variable */
 	name: string;
 }
 
+/**
+ * This corresponds directly to the block config.
+ */
 interface BlockConfig {
+	/** The available bindings for the block */
```

Comment:
Nitpick: I love code comments, but many of the added comments here (including this line) don't add much value and slow down the reader. They aren't adding any additional context, they just rearrange the existing semantic tokens into a sentence. "The {property name} for the {instance name}" is already obvious and inferable.

Reply:
That's a fair point. I wanted to use this as an opportunity to start the comments, and have them be built over time. Instead, I'll take out most of em and just leave the ones that make sense in place.